### ISSUE TITLE
Midterm Peer Review Xinyue Liu

### ISSUE THREAD

**Issue by AmysnL** [manuhanu/Predict-Bitcoin#6]
The objective of the project is to predict the price of Bitcoin and classify the price as up or down. They apply the OLS regression, Ridge regression and the Logistic model in the EDA. The data they use is the resampled minute level Bitcoin price data.

The histograms and scatterplot along with brief explanations clearly visualizes the relationship among variables and give the audience a general sense of the whole picture. However, from my perspectives, the histograms could be filtered to be shown that it may be better to select some signature ones that could be explained better and in greater details, which could be a more reasonable logic to organize the structure.

It is very clear that they have a table of the three models along with the accuracy. Also, the approach to identify the overfitting problems by trying to separately fit the logistical regression is impressive. The future plan is also very logically elaborated, especially they point out the cross-validation approach to resolve the limited data points problem.

One thing I may take into consideration is the training set and testing set split. It may be a problem if the year matters, so that the testing set may perform differently not because of a bad model but other factors.
